Dockerfile

```diff
@@ -27,10 +28,13 @@ ENV PYTHONPATH="$GRAPHITE_ROOT/lib:$GRAPHITE_ROOT/webapp" \
     PATH="$PATH:$GRAPHITE_ROOT/bin"
 WORKDIR $GRAPHITE_ROOT
 
+COPY ./carbon.conf.example conf/carbon.conf.example
+COPY ./render-carbon-template.sh conf/render-carbon-template.sh
+
 # Set up basic config
 RUN cp conf/graphite.wsgi.example webapp/graphite/wsgi.py && \
-    cp conf/carbon.conf.example conf/carbon.conf && \
-    cp conf/storage-schemas.conf.example conf/storage-schemas.conf
+    cp conf/storage-schemas.conf.example conf/storage-schemas.conf && \
+    ./render-carbon-template.sh
```

Comment:
So you're rendering here to generate the `carbon.conf` file? Do you not need to render later as well, like in the entrypoint script or something?